dashboard/test/models/census/census_your_school2017v6_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class Census::CensusYourSchool2017v6Test < ActiveSupport::TestCase
+  test "basic census 2017 v6 submission" do
+    submission = build(:census_your_school2017v6)
+    assert submission.valid?, submission.errors.full_messages
+  end
+
+  test "census 2017 v6 submission missing topic_ethical_social" do
+    submission = build(:census_your_school2017v6, :requiring_followup, topic_ethical_social: nil)
+    assert_not submission.valid?, submission.errors.full_messages
```

Comment:
This can probably just be `refute submission.valid?` In the case where the assertion fails, the submission is valid so using the error messages as the assertion message won't actually tell you anything.

Reply:
Good point.  I see this in a lot of other places.. do they have same issue?  e.g. https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/models/census/census_submission_test.rb

Reply:
Ah, I must have written those before I knew about `refute`. It isn't a problem per se, just that logging the messages on an assertion failure doesn't provide any extra info.